tests/propTests/disabledPropTest.js

```diff
@@ -0,0 +1,18 @@
+import { within } from '@testing-library/react';
+
+export const disabledPropTest = [
+  [
+    { disabled: true },
+    (rootElement) => {
+      expect(rootElement).toHaveClass('isRootDisabled');
```

Comment:
As mentioned somewhere earlier, the native element is sometimes `disabled`, sometimes there is a class name, and sometimes it's both. It may be useful to make all these cases reusable.

Reply:
see above: https://github.com/react-ui-org/react-ui/pull/301#discussion_r708131872